docs/StackedFilters.md

```diff
@@ -347,36 +352,38 @@ import {
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 
 const postListFilters: FiltersConfig = {
+    id: textFilter({ operators: ['eq', 'neq'] }),
     title: textFilter(),
-    views: numberFilter(),
+    published_at: dateFilter(),
+    is_public: booleanFilter(),
     tag_ids: referenceFilter({ reference: 'tags' }),
-    published: booleanFilter(),
 };
 
-const PostList = () => (
-    <ListBase>
-        <Accordion sx={{ my: 1 }}>
+const PostListFiltersForm = () => {
+    const { filterValues } = useListContext();
+    return (
+        <Accordion>
             <AccordionSummary
                 expandIcon={<ExpandMoreIcon />}
                 aria-controls="filters-content"
                 id="filters-header"
             >
-                <Typography>Filters</Typography>
+                <Typography>
+                    {Object.keys(filterValues).length ? (
+                        <>
+                            {Object.keys(filterValues).length} filter(s) applied
+                        </>
```

Comment:
use a template instead to make it fit in one line

Reply:
This snippet comes from Storybook, I did a naive copy/paste 😅